lib/bumper.js

```diff
@@ -101,15 +84,13 @@ class Bumper {
       return Promise.resolve()
     }
 
-    return this._getOpenPrScope()
-      .then((scope) => {
-        logger.log(`Found a ${scope} bump for the current PR`)
+    return this._getOpenPrInfo()
```

Comment:
Ah, info is the total wrapper for all data in the PR description and scope is the semver flag?